tractor/_actor.py

```diff
@@ -171,13 +176,16 @@ class Actor:
     _root_n: Optional[trio.Nursery] = None
     _service_n: Optional[trio.Nursery] = None
     _server_n: Optional[trio.Nursery] = None
+    _lifetime_stack: ExitStack = ExitStack()
```

Comment:
Oh right, we might want tests for this :rofl:..

Maybe it's ok to just slip it in for now and worry later?